testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1401,8 +1401,7 @@ public void testAcidUpdateWithSubqueryAssignment()
         // TODO support UPDATE with correlated subquery in assignment
         withTemporaryTable("test_update_subquery", true, false, NONE, tableName -> {
             onTrino().executeQuery(format("CREATE TABLE %s (column1 INT, column2 varchar) WITH (transactional = true)", tableName));
-            onTrino().executeQuery(format("INSERT INTO %s VALUES (1, 'x')", tableName));
-            onTrino().executeQuery(format("INSERT INTO %s VALUES (2, 'y')", tableName));
+            onTrino().executeQuery(format("INSERT INTO %s VALUES (1, 'x'), (2, 'y')", tableName));
```

Comment:
@Praveen2112 do you understand why INSERT-ing two rows at once avoids https://github.com/trinodb/trino/issues/8268 while two separate INSERTs followed up UPDATEs lead to a failure on Hive side?

Reply:
Still investigating on why it works.

Reply:
I think this is why it works - https://github.com/trinodb/trino/issues/8268#issuecomment-863995258

Reply:
When we run the inserts in a same query - single delta file is created and delete information are properly mapped to the file (since they have different rowId) thus it works.